ZenWithTerms/tla/ZenWithTerms.tla

```diff
@@ -29,6 +29,7 @@ VARIABLE descendant
 \* Values to bootstrap the cluster
 VARIABLE initialConfiguration
 VARIABLE initialValue
+VARIABLE initialAcceptedVersions
```

Comment:
why the `s` at the end? we're not doing this for the other maps either. Perhaps just add a comment that this is a map.

Reply:
Habit. Fixed in 84f11c283ab88a868fffad08c4360da54bba8e8e